### PR TITLE
Add optional test result column for list view

### PR DESCRIPTION
**Pull request by AndrewGnagy**
This will make an optional column available for list views when using the TestNG plugin.

latesttestngcol

Format is:
{Total Failed Tests}({Difference From Previous Build}) failed/{Total Tests Run} total
ie. "1(+1) failed/102 total"

testngresultcolumn

### CODE REVIEW

**Comment by jvhaggard** [comment]
@Vlatombe There are a couple open PRs on this plugin that is up for adoption and we want to stop running our own internal custom plugin.  What is the review process to get them merged?  We're talking internally about maybe adopting the plugin but the group is still discussing it.

File: src/main/java/hudson/plugins/testng/ResultColumn.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+         return false;
+      }
+
+      @Override
+      public String getDisplayName() {
+         return "Latest TestNG Results";
+      }
+   }
+
+   @Extension
```
Comment (Vlatombe) [inline src/main/java/hudson/plugins/testng/ResultColumn.java]: Just annotate the descriptor class with `@Extension`

**Comment by Vlatombe** [comment]
@jvhaggard I have no idea why you pinged me, but I can only point you to the [adoption process](https://wiki.jenkins.io/display/JENKINS/Adopt+a+Plugin).

**Comment by AndrewGnagy** [comment]
Outdated

**Pull request closed without merging**